Hush/Assets/Scripts/Keys/KeyCollect.cs

```diff
@@ -0,0 +1,23 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class KeyCollect : MonoBehaviour
+{
+
+    private static int numOfKeys = 0;
+    public static int NumOfKeys {
+        get {
+            return numOfKeys;
+        }
+    }
+    public static int keys = 0;
+
+    void OnTriggerEnter(Collider collider) {
+        if (collider.gameObject.name == "Player") {
+            numOfKeys += 1;
+            gameObject.SetActive(false);
+            //Destroy (gameObject);
```

Comment:
Right now I have it so that they key just disappears from sight, but once we have the actual key spawner with their locations this can be uncommented and Line 19 can be removed. This is just for demo purposes of the function.

Reply:
Spawner exists in main, it should be easy enough to merge your changes with main

Reply:
Yeah just saw that and I added it to there